### DISCUSSION TITLE
Other variants now obsolete?

### DISCUSSION THREAD

**Discussion by Anagastes** [aristocratos/btop#79] [General]
It's not quite clear which of your implementations are now being actively developed :) 

I guess the C++ variant is now your preferred version.

I would recommend to make it clear for the bash and the python variant as well (bash already refers to the python variant).

**Comment by aristocratos** [comment]
@Anagastes 
I don't know if they're obsolete really, you could say they're kind of compatibility fallbacks for each other.
If `btop++` fails to compile or don't work on your platform you can try `bpytop` and if both `btop++` fails and you don't have `python 3.6>` available, you can try `bashtop`.

`bashtop` is probably not going to get any big updates anymore (unless other people wanna keep working on it).
I've planned to try to backport some of the new features in `btop++` to `bpytop`, when I've got time (and if feasable).

There seem to be plenty of people using all the variants, so I'm not gonna try to force anyone to change, just hint that there are better alternatives :)
